client/lib/user/support-user-interop.js

```diff
@@ -36,11 +36,11 @@ const reduxStoreReady = new Promise( ( resolve ) => {
 } );
 export const setReduxStore = _setReduxStore;
 
-/**
- * Check if there's a support user to be activated on boot
- * @return {bool} true if a support user token is waiting to be injected on boot, false otherwise
- */
-export const shouldBoot = () => {
+// Evaluate isSupportUserSession at module startup time, then freeze it
```

Comment:
oh, right. I thought It was the block comment for `_isSupportUserSession` function.